CMakeLists.txt

```diff
@@ -607,6 +608,7 @@ if(SLANG_ENABLE_TESTS)
         EXCLUDE_FROM_ALL
         INCLUDE_FROM_PRIVATE slang
         FOLDER test
+        INCLUDE_DIRECTORIES_PUBLIC include
```

Comment:
The INCLUDE_DIRECTORIES in redundant in the presence of this INCLUDE_FROM

Reply:
I tried to add `include` to `INCLUDE_FROM_PRIVATE`, but it doesn't work.
It fails at
```
CMake Error at cmake/SlangTarget.cmake:223 (target_include_directories):
  Error evaluating generator expression:

    $<TARGET_PROPERTY:/home/zhangkai/Documents/slangwork/slang/include,INTERFACE_INCLUDE_DIRECTORIES>
```

So I just add a 
`INCLUDE_DIRECTORIES_PRIVATE include`